docs/reference/search/search-template.asciidoc

```diff
@@ -604,7 +604,7 @@ query as a string instead:
 
 The `{{#url}}value{{/url}}` function can be used to encode a string value
 in a HTML encoding form as defined in by the 
-http://www.w3.org/TR/html4/[HTML specification].
+https://www.w3.org/TR/html4/[HTML specification].
```

Comment:
Hmm... I wonder if we should update this to https://html.spec.whatwg.org/? I'll open another issue for that.
```suggestion
https://www.w3.org/TR/html4/[HTML specification].
```